WIP: Add [system:abstract] to concept, reference, procedure templates (#130)
redhat-documentation/modular-docs: modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

//
// <List assemblies here, each on a new line>

////
Base the file name and the ID on the module title. For example:
* file name: proc-doing-procedure-a.adoc
* ID: [id="doing-procedure-a_{context}"]
* Title: = Doing procedure A
////

>  The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken.

Reviewer: The line starts with a stray space character.

>  The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.

Reviewer 2: ```suggestion
The ID is an anchor that links to the module. Do not change it after you publish the module to avoid breaking existing links.
```

Author: OK. Updated the first part. But some groups do regularly change the IDs so we need to say avoid.

////

[id="proc-doing-one-procedure_{context}"]

////

> The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.

Reviewer 2: ```suggestion
The `context` attribute enables module reuse. Every module ID includes {context}, which ensures that the module has a unique ID so you can include it multiple times in the same guide.
```

Author: Updated

////

= Doing one procedure
////
Start the title of a procedure module with a verb, such as Creating or Create. See also _Wording of headings_ in _The IBM Style Guide_.
////

[role="_abstract"]

> A short introductory paragraph that provides an overview of the module is required. Procedure modules should include both what users will do and why users want to do it.

Reviewer 2: ```suggestion
A short introductory paragraph that provides an overview of the module. Procedure modules should include the steps that users perform and address user motivation.
```

Author: updated

. Use an unnumbered bullet (*) if the procedure includes only one step.

.Verification steps
////

> Optional. Delete if not applicable. Provide the user with verification methods for the procedure, such as expected output or commands that can be used to check for success or failure.

Reviewer 2: ```suggestion
Delete this section if it does not apply to your module. Provide the user with verification methods for the procedure, such as expected output or commands that confirm success or failure.
```

Author: Updated

////

. Start each step with an active verb.

> . Include one command or action per step.

Reviewer 2: I would advocate for one "non-trivial" command per step. For example, if your procedure is for some kind of GUI form or wizard and users click a "Next" button to proceed to the next step, that doesn't need to be a separate step. 

Compare the difference:

. Do this thing.
. Select "Next".
. Do this other thing.
. Select "Next".

VS 

. Do this thing and then select "Next".
. Do this other thing and then select "Next".

> . Use an unnumbered bullet (*) if the procedure includes only one step.

Reviewer 2: ```suggestion
. Use an unordered list item (*) if the procedure includes one step only.
```

Author: AFAIK 'only one step' is correct.
